### ISSUE TITLE
Strange usage of learning rate 

### ISSUE THREAD

**Issue by jackaceuser** [sunset1995/pytorch-layoutnet#1]
Hi

Thank you for the implementation. It's very helpful.

I was wondering why do you start with very low learning rate (1e-6 at warmup) and then change to 0.018 later.

Isn't that supposed to be other way around? Starting with high learning-rate and then lowering it?

**Comment by sunset1995** [comment]
This is a training trick called "warmup". Empirically, this trick help convergence and stabilize training.  
Here are some papers exploit this trick:
- [YOLO](https://arxiv.org/abs/1506.02640)
- [Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour](https://arxiv.org/abs/1706.02677)
- [MegDet: A Large Mini-Batch Object Detector](https://arxiv.org/abs/1711.07240)

But actually I have not compared the result w/ or wo/ this trick. It could be done by setting the `--warmup_epochs` arguments to `0` in `train.py`

Hope my answer help :)
